app/views/foreman_leapp/job_templates/leapp_preupgrade.erb

```diff
@@ -13,15 +13,20 @@ template_inputs:
   required: true
   default: "ga"
   options: "ga\ne4s\neus\naus"
+- name: Target
+  description: "Set the target OS if not the last release"
```

Comment:
```suggestion
  description: "Set the minor version of the target operating system unless you aim for the latest release"
```
If it works, please use the same for `upgrade` below.
Feel free to suggest an alternative wording if I guessed it wrong.

Reply:
Perhaps we should also tell the user what the input is supposed to look like? `X.Y`?

Reply:
if we stick to leapp help should be "Specify RHEL version to upgrade to for default detected upgrade flavour"
is this ok ?

Reply:
"Specify RHEL version to upgrade to" makes sense.
But "for default detected upgrade flavour" confuses me - can you please elaborate? What does it mean? Flavour of what? Why do I have to specify something when it's "detected"?

Reply:
How about this?
```suggestion
  description: "Specify the release version of the operating system to upgrade to. If you do not specify, the latest release will be used."
```

Reply:
Sounds good for me.
thanks for the suggestion @Lennonka